src/components/Footer/Footer.js

```diff
@@ -62,7 +64,7 @@ const Footer = () => {
           className={classes.link}
         >
           <i className={`fas fa-vote-yea ${classes.linkIcon}`}></i>
-          <span>vote</span>
+          <span>{t('vote')}</span>
```

Comment:
Same as above, this being a a link to a subdomain, a translation might not be needed